Assets/GothicVR/Scripts/SO Template/DateTimeGameEvent.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using UnityEngine;
+using UnityEngine.Events;
+
+[CreateAssetMenu(menuName = "Events/DateTime Event Channel")]
+
+public class DateTimeGameEvent : ScriptableObject
+{
+    public UnityAction<DateTime> OnEventRaised;
```

Comment:
There's a more detailed comment below. But in short: Is this abstraction really needed with the additional Event classes? Why not leveraging Unity's events directly?